src/test/java/teammates/test/cases/webapi/SearchStudentsActionTest.java

```diff
@@ -0,0 +1,165 @@
+package teammates.test.cases.webapi;
+
+import org.junit.Test;
+import teammates.common.datatransfer.DataBundle;
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.common.util.Const;
+import teammates.ui.webapi.action.JsonResult;
+import teammates.ui.webapi.action.SearchAccountsAction;
+import teammates.ui.webapi.action.SearchStudentsAction;
+import teammates.ui.webapi.output.AdminSearchResultData;
+import teammates.ui.webapi.output.SearchStudentsData;
+
+/**
+ * SUT:{@link SearchStudentsAction}
+ */
+public class SearchStudentsActionTest extends BaseActionTest<SearchStudentsAction> {
+
+    @Override
+    protected void prepareTestData() {
+        DataBundle dataBundle = getTypicalDataBundle();
+        removeAndRestoreDataBundle(dataBundle);
+        putDocuments(dataBundle);
+    }
+
+    @Override
+    protected String getActionUri() {
+        return Const.ResourceURIs.STUDENTS_SEARCH;
+    }
+
+    @Override
+    protected String getRequestMethod() {
+        return GET;
+    }
+
+    @Override
+    @Test
+    protected void testExecute() {
```

Comment:
It's probably good practice to split this method up into individual test methods with the proper test method naming convention.

Reply:
idk also, seems like teammates convention to do this for actions though

Reply:
@xpdavid is there a specific convention to follow?

Reply:
Prefer individual test methods 👍

Reply:
fixed